README.md

```diff
@@ -64,6 +64,12 @@ See our [Getting Started Guide](src/docs/getting-started.md) in the docs.
 
 See the [Projects](https://github.com/Azure/missionlz/projects) page for the release timeline and feature areas.
 
+Here's what the repo consists of as of April 2021 (items in grey are either planned but not yet implemented by Mission LZ, or are reference products or resources that could fill a customer workload need):
```

Comment:
Most of the content is in some form of gray, which makes this sentence confusing. I can tell what you mean - there are things that are gray and missing definition, like the workstations. I'm not sure how to describe this on the image. We could leave the image as-is and remove the statement.

Reply:
😄 that makes sense. I've updated the graphic to include labels for those out-of-scope things with "Bring your own <X>" and have removed the bit about color of the object.